File: Login/login_sys.go
package main

import "fmt"

func main(){
	// for i := 0; i < 5;i++{
	// 	fmt.Printf("5 x %v = %v\n",i,5*i)
	// }
	for {
		var name string
		var age int
		var passwr string
		var gender string
		fmt.Print("Enter your name: ")
		fmt.Scan(&name)
		fmt.Print("Enter your age: ")
		fmt.Scan(&age)
		fmt.Print("Enter your gender: ")
		fmt.Scan(&gender)
		fmt.Print("Enter your password: ")
		fmt.Scan(&passwr)
		var confr string
		fmt.Print("Confirm your password: ")
		fmt.Scan(&confr)
		if (passwr == confr){
			fmt.Println("Account created successful")
			println("\n\n\n")
			fmt.Printf("#####   ACCOUNT DETAILS ########\nNAME : %v\nAGE : %v\nGENDER : %v\n",name,age,gender)
			break
		}else{
			fmt.Println("Password mismatch")
			fmt.Println("Please try again")
		}
	}
}

File: Login/login_sys.rs
use std::io;

fn main(){
    let mut name = String::new();
    let mut age = String::new();
    let mut gender = String::new();
    let mut password = String::new();
    let mut confirm = String::new();
    println!("Enter your name: ");
    io::stdin().read_line(&mut name).expect("failed to read line");
    println!("How old are you: ");
    io::stdin().read_line(&mut age).expect("failed to read line");
    // loop{
    //     if age == "18"{
    //         println!("Too young to sign in right now");
    //     }else{
    //         break;
    //     }
    // }
    println!("What's your gender: ");
    io::stdin().read_line(&mut gender).expect("failed to read line");
    loop{
        println!("Enter your password: ");
        io::stdin().read_line(&mut password).expect("failed to read line");
        println!("confirm your password: ");
        io::stdin().read_line(&mut confirm).expect("failed to read line");
        if password == confirm{
            println!("Account ceated succesfully");
            println!("\n\n");
            println!("#####   ACCOUNT DETAILS ########NAME : {}AGE : {}GENDER : {}",name,age,gender);
            break;
        }else{
            println!("Password mismatch");
        }
    }
}